src/plugins/inspector/public/views/requests/index.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { i18n } from '@kbn/i18n';
+
+import { RequestsViewComponent } from './components/requests_view';
+import { Adapters, InspectorViewDescription } from '../../types';
+
+export const RequestsView = {
+  title: i18n.translate('inspector.requests.requestsTitle', {
+    defaultMessage: 'Requests',
+  }),
+  order: 20,
+  help: i18n.translate('inspector.requests.requestsDescriptionTooltip', {
+    defaultMessage: 'View the requests that collected the data',
+  }),
+  shouldShow(adapters: Adapters) {
+    return Boolean(adapters.requests);
+  },
+  component: RequestsViewComponent,
+} as InspectorViewDescription;
```

Comment:
Is casting necessary here, or could you instead do `export const RequestsView: InspectorViewDescription`?

Reply:
agree, fixed